robottelo/vm.py

```diff
@@ -375,10 +375,11 @@ def configure_rhai_client(self, activation_key, org, rhel_distro):
         service.
 
         :param activation_key: Activation key to be used to register the
-        system to satellite
+            system to satellite
         :param org: The org to which the system is required to be registered
         :param rhel_distro: rhel distribution for
         :return: None
+
```

Comment:
You are right.  A blank line is a must before before the input parameters section but not after.